src/main/resources/kibana-rest.yml

```diff
@@ -62,3 +62,43 @@ kibana_user:
   versions:
     ">= 7.6.0": "/internal/security/me"
     ">= 5.0.0 < 7.6.0": "/api/security/v1/me"
+
+kibana_security_exception_list:
+  versions:
+    ">= 7.10.0": "/api/exception_lists/_find?namespace_type=agnostic"
+
```

Comment:
Also, the call below will return the individual exception items associated with rules including logical conditions applied. ([API docs](https://www.elastic.co/guide/en/security/current/exceptions-api-find-exception-items.html)). We would have to first fetch all the exception list ids and then loop over them to fetch the exception list items for each list. Is this possible to do in the support diagnostics framework?
```
GET api/exception_lists/items/_find?list_id=<list-id>
```